src/promptflow/promptflow/batch/_batch_engine.py

```diff
@@ -128,6 +128,8 @@ def run(
         run_id: Optional[str] = None,
         max_lines_count: Optional[int] = None,
         raise_on_line_failure: Optional[bool] = False,
+        resume_from_run_storage: Optional[AbstractRunStorage] = None,
```

Comment:
Please remember to add docstring for these two param

Reply:
Good catch. Done.